### ISSUE TITLE
modeler insert segmented Spline

### ISSUE THREAD

**Issue by boyang2022** [pyansys/pyaedt#952]
tried the following: 

```python
from pyaedt.desktop import Desktop
from pyaedt import Q3d
from pyaedt.modeler.Primitives import PolylineSegment

d = Desktop("2022.1", False, False)
q3d = Q3d(designname="test", specified_version="2022.1")
q3d.modeler.model_units = "mm"
q3d.modeler.delete()

point = [[1., 1., 0], [1., 2., 1.], [1., 3., 1.]]

line1 = q3d.modeler.create_polyline([[0, 0, 0], point[0]])

line1.insert_segment(position_list=point,
                     segment=PolylineSegment("Spline", num_points=3, num_seg=3))

```
And insert part is not segmented.
![image](https://user-images.githubusercontent.com/93125854/158170599-762bace8-cb85-4c4d-88ab-09d750cec197.png)

Need to insert segmented Spline.

in PolylineSegment spine has no num_seg
![image](https://user-images.githubusercontent.com/93125854/158171020-f55d6902-7f7f-42b6-98f1-239f3e21374d.png)

**Comment by MaxJPRey** [comment]
Hi @boyang2022 ,

Following is an example that should work and create the spline as you are expecting it to be.

```
from pyaedt.desktop import Desktop
from pyaedt import Q3d
from pyaedt.modeler.Primitives import PolylineSegment

d = Desktop("2022.1", False, False)
q3d = Q3d(designname="test", specified_version="2022.1")
q3d.modeler.model_units = "mm"
q3d.modeler.delete()

point = [[1., 1., 0], [1., 2., 1.], [1., 3., 1.]]

q3d.modeler.create_polyline(position_list=point, segment_type=PolylineSegment("Spline", num_points=3, num_seg=3))
```

Please let me know if that solves the issue you faced.

Best Regards.

**Comment by boyang2022** [comment]
hello @MaxJPRey thanks for your reply.

Yes, I can create a spline. However, I want is to **insert** a segmented spline after a straight line. For example,
![image](https://user-images.githubusercontent.com/93125854/158528233-9e88e69f-99e9-446d-b300-c73958602f70.png)

**Comment by MaxJPRey** [comment]
Hi @boyang2022 ,
This will be corrected in the next release (v0.4.46).
Thanks.
